src/overlays/actors/ovl_En_Bji_01/z_en_bji_01.h

```diff
@@ -2,12 +2,31 @@
 #define Z_EN_BJI_01_H
 
 #include <global.h>
+#include "../ovl_Obj_Moon_Stone/z_obj_moon_stone.h"
```

Comment:
We don't use relative includes.
```suggestion
#include "overlays/actors/ovl_Obj_Moon_Stone/z_obj_moon_stone.h"
```